src/ballet/chacha20/README.md

```diff
@@ -0,0 +1,15 @@
+# fd_chacha20_rng
+
+## Testing (MacOS)
+
+Build binary by using the following command
+
+```
+gcc -o test_fd_chacha20_rng test_fd_chacha20_rng.c fd_chacha20_rng.c -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -lcrypto
+```
```

Comment:
Adding a README is nice, but the normal way to build a test is with `make`. Does this build with `make`? If so, you should find the unit test in the unit-tests directory.

Reply:
The unit test should also probably be called `test_chacha20` without the `_rng` suffix.

Reply:
All renaming will be done once code from `openssl` is imported.

Regarding the `make` file, I need to get my feets wet on that 👀

Reply:
just added a Makefile and updated the README docs ✅